api/base/src/main/java/org/geysermc/api/session/Connection.java

```diff
@@ -55,5 +55,12 @@
      */
     String xuid();
 
-
+    /**
+     * Transfer the connection to a server. The current server is a valid target.
```

Comment:
`The current server is a valid target.` seems a bit confusing. Maybe "A Bedrock player can successfully transfer to the same server they are currently playing`?